iexec-sms-library/build.gradle

```
@@ -5,14 +5,20 @@ plugins {
     id 'maven-publish'
 }
 
+ext {
+    springBootVersion = '3.0.13'
+}
+
 dependencies {
+    implementation platform("org.springframework.boot:spring-boot-dependencies:${springBootVersion}")
```

In the same way, I think I would prefer to keep
```suggestion
    implementation platform('org.springframework.boot:spring-boot-dependencies:3.0.13')
```
in order to stay consistent with other repositories.
In my opinion, this kind of change would be interesting if there was a way to define a a single `springBootDependencies` and to use it in all gradle (sub)projects of the repository.

---

Resolved in [52286dc](https://github.com/iExecBlockchainComputing/iexec-sms/pull/285/commits/52286dc3d3a52e6b40854025aa76db758489a310)

---

The root `build.gradle` has been updated, the `iexec-sms-library/build.gradle` has not.